src/Compilers/VisualBasic/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1897,13 +1897,13 @@
   <Node Name="BoundInterpolatedStringExpression" Base="BoundExpression" HasValidate="true">
     <!-- Type is required for this node type; may not be null -->
     <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
-    <Field Name="Contents" Type="ImmutableArray(Of BoundNode)"/>
+    <Field Name="Contents" Type="ImmutableArray(Of BoundExpression)"/>
     <Field Name="Binder" Type="Binder" Null="disallow" />
   </Node>
   
   <!-- There is no BoundInterpolatedStringText because we use BoundLiteral of type string for this purpose. -->
   
-  <Node Name="BoundInterpolation" Base="BoundNode">
+  <Node Name="BoundInterpolation" Base="BoundExpression" HasValidate="true">
```

Comment:
They aren't expressions. We could add some base types and make the text content an explicit type but I think it would be dangerous to imply these are always expressions. It's more of an implementation shortcut that I reused BoundLiteral there.